mavlink_vehicles/apm.hh

```diff
@@ -17,8 +17,8 @@
 #pragma once
 
 enum APM_CUSTOM_MODE {
-    APM_CUSTOM_MODE_GUIDED = 3,
-    APM_CUSTOM_MODE_AUTO = 4,
-    APM_CUSTOM_MODE_BRAKE =17
+    APM_CUSTOM_MODE_GUIDED = 4,
+    APM_CUSTOM_MODE_AUTO = 3,
```

Comment:
Yes. I've swapped it unintentionally when moving those numbers to the enum. See [this](https://github.com/ArduPilot/ardupilot/blob/f0f432250060fba031258988ec91166249cb4723/ArduCopter/defines.h#L92).